src/Tooltip/Tooltip.component.tsx

```diff
@@ -0,0 +1,255 @@
+// REACT
+import * as React from 'react';
+// VENDOR
+import classNames from 'classnames';
+import styled, { css } from '@xstyled/styled-components';
+import { th } from '@xstyled/system';
+
+export type Position =
+    | 'topStart'
+    | 'top'
+    | 'topEnd'
+    | 'rightStart'
+    | 'right'
+    | 'rightEnd'
+    | 'bottomEnd'
+    | 'bottom'
+    | 'bottomStart'
+    | 'leftEnd'
+    | 'left'
+    | 'leftStart';
+
```

Comment:
I don't envy you having to come up with these names. At first I had no clue what they meant until I played around knobs, then it made perfect sense. We should consider reusing these names for all components that have some type of positioning scheme to be consistent.

Reply:
I agree. I'll be abstracting and applying this logic to DropDown.